ddtrace/settings/config.py

```diff
@@ -209,6 +221,9 @@ def __init__(self):
         self._raise = asbool(os.getenv("DD_TESTING_RAISE", False))
         self._trace_compute_stats = asbool(os.getenv("DD_TRACE_COMPUTE_STATS", False))
         self._appsec_enabled = asbool(os.getenv("DD_APPSEC_ENABLED", False))
+        self._obfuscation_query_string_pattern = re.compile(
+            os.getenv("DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN", DD_TRACE_OBFUSCATION_QUERY_STRING_PATTERN_DEFAULT)
```

Comment:
we should handle when the env variable value is an empty string/disabled.

Reply:
we are also missing tests for setting this env variable

Reply:
Done!